irods/test/meta_test.py

```diff
@@ -121,6 +122,18 @@ def tearDown(self):
 
     from irods.test.helpers import create_simple_resc_hierarchy
 
+    def test_atomic_metadata_json_error__issue_606(self):
+        path = self.coll_path + "/atomic_meta_issue_606"
+        obj = self.sess.data_objects.create(path)
+        obj.unlink(force = True)
+        try:
+            obj.metadata.apply_atomic_operations(AVUOperation(operation="add", avu=iRODSMeta('a','b','c')))
+        except ex.iRODSException as e:
+            resp = e.server_msg.get_json_encoded_struct()
```

Comment:
This needs to be documented in the README if users are expected to do this.

How does the user know which APIs support this?

Reply:
User will have to read docs generated from the APIs.  I think this is all in web pages like https://docs.irods.org/4.3.2/doxygen/replica__truncate_8h.html but  don't know how to point users to the "general set" of such API pages.  Perhaps others have a suggestion for that, for the README. (@korydraughn @alanking @trel)

Reply:
Oh, it seems you already did this in the "README addition" commit. Please ignore my previous comment.

Reply:
The change to address this is good.

Marking this as resolved.